<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow
{
background-color:#FFEEE7;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">

QTabWidget:pane {
border:4px;
background-color:#FFEEE7;
border-color:#666666;
border-radius:4px;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Publicite</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>251</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Publicite</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Durée</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>date d'ajout</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>19</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>12</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>411</width>
        <height>192</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Durée</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date d'ajout</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prix</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>272</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>272</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>60</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Reclmation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>231</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter reclamation</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Produit ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Date d'ajout</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
       </property>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit
{
color:#000;
border:2px solid  #999999; 
  border-radius: 4px;

}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>411</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Produit ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date d'ajout</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>260</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Exporter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>120</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
 background-color:#e86054;  /* Green */ 
  border: none;
 /* color: gray; 
  padding: 15px 32px;*/
  text-align: center;
  font-size: 16px;
	border:9px;
}
QPushButton:hover{

background-color:#e0987b;
color:#7d2b20;
}
QPushButton:click{

background-color:#FCDBCE;
}</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
